applications/ChimeraApplication/chimera_application_variables.h

```diff
@@ -29,15 +29,18 @@
 namespace Kratos
 {
 
-KRATOS_DEFINE_VARIABLE(double, CHIMERA_DISTANCE);
-KRATOS_DEFINE_VARIABLE(double, ROTATIONAL_ANGLE);
-KRATOS_DEFINE_VARIABLE(double, ROTATIONAL_VELOCITY);
+KRATOS_DEFINE_APPLICATION_VARIABLE(CHIMERA_APPLICATION,double, CHIMERA_DISTANCE);
+KRATOS_DEFINE_APPLICATION_VARIABLE(CHIMERA_APPLICATION,double, ROTATIONAL_ANGLE);
+KRATOS_DEFINE_APPLICATION_VARIABLE(CHIMERA_APPLICATION,double, ROTATIONAL_VELOCITY);
 KRATOS_DEFINE_3D_APPLICATION_VARIABLE_WITH_COMPONENTS(CHIMERA_APPLICATION, ROTATION_MESH_DISPLACEMENT);
 KRATOS_DEFINE_3D_APPLICATION_VARIABLE_WITH_COMPONENTS(CHIMERA_APPLICATION, ROTATION_MESH_VELOCITY);
 
-KRATOS_DEFINE_FLAG( FS_CHIMERA_VELOCITY_CONSTRAINT);
-KRATOS_DEFINE_FLAG( FS_CHIMERA_PRESSURE_CONSTRAINT);
-KRATOS_DEFINE_FLAG( CHIMERA_INTERNAL_BOUNDARY);
+class ChimeraFlags {
```

Comment:
thats also what I do in Mapping

Reply:
'How to access these flags from python then ? will a normal 
```python
import KratosMultiphysics.ChimeraApplication as chm 
chm.CHIMERA_INTERNAL_BOUNDARY 
```

work  ? Because I am getting error that chm has no variable `CHIMERA_INTERNAL_BOUNDARY`

Reply:
well you need to expose them. I do it like this but this is just an example:
https://github.com/KratosMultiphysics/Kratos/blob/master/applications/MappingApplication/custom_python/add_custom_mappers_to_python.cpp#L160-L166

Reply:
It is exposed ! 

https://github.com/KratosMultiphysics/Kratos/blob/bb1fa325a12d028da22b062818b7eb98ec9c4611/applications/ChimeraApplication/custom_python/chimera_python_application.cpp#L44

Reply:
hmm then I don't know
Maybe with all those changes you need a clean build?

Reply:
looking at [this](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/includes/define_python.h#L93) I think the name is now `ChimeraFlags::CHIMERA_INTERNAL_BOUNDARY`

Probably you cannot use the macro then and do it manually (like I do it)
It is only one line...

Reply:
@roigcarlo I guess if this is the case then maybe we could have a second macro where one can also pass the name?

Reply:
@adityaghantasala can you please check if it works with the full name?

Reply:
@philbucher  It is a clean build ... 
I am testing now ... but I do not think it will work. because python does not have `::` operator.

Reply:
No it does not work ... 

I am explicitly adding the flag to the module (without using the macro) then. I tried, it works.

Reply:
hm ok

Then I guess we could have a second macro

Reply:
It will be a different PR and we can adapt this part in that PR ?

Reply:
#6670